app/shared/utilities/localStorageService.js

```diff
@@ -14,7 +14,9 @@ eShopApp.factory('$localStorage', function ($window) {
         $window.localStorage[key] = JSON.stringify(value);
     },
     isExists: (key) => {
-        return !!$window.localStorage[key];
+        if($window.localStorage[key])
```

Comment:
you can simplify this method by returning !!$window.localStorage[key]

Reply:
I was doing that at first as in previous commit but it was giving me errors, so I replaced it but I'll refactor the code tomorrow isa